src/components/TokenPairSelector/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TokenPairSelectorList } from 'components/TokenPairSelectorList'
+
+import ArrowWhite from 'assets/img/arrow-white.svg'
+
+const TokenPairSelectorStyled = styled.div`
```

Comment:
ubernit: We usually call the outermost styled element `wrapper` (instead of `TokenPairSelectorStyled ` in this case). It's just a convention we've used. Open for discussion if it's not good. In my eyes, I like it. Wdyt?

Reply:
Yeah sounds good. So in this case `TokenPairSelectorStyled` is only used in this index.tsx file, so then just calling it `wrapper` would be a consistent approach of naming it, correct?

Reply:
Updated.